docs/localized/KR/docs/Installation.md

```diff
@@ -16,7 +16,7 @@ Clone(복제)하고 추가종속성을 가지는 Python(파이썬)을 설치해
 </p>
 
 ## Windows 사용자
-Windows에서 환경을 설정하기 위해, [세부 사항](Installation-Windows.md)에 설정 방법에 대해 작성하였습니다.
+Windows에서 환경을 설정하기 위해, [세부 사항](Installation-Anaconda-Windows.md)에 설정 방법에 대해 작성하였습니다.
```

Comment:
FWIW we've been holding off changing any of the translated docs, since it usually screws up some character encoding. This looks OK though.

Reply:
I considered that but figured it'd be OK with this change.